src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,43 @@
+.clock {
+  position: relative;
+  height: $clock-height;
+  width: $clock-width;
+  border: 4px dotted navy;
+  border-radius: 50%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: chocolate;
+
+  &__center {
+    position: relative;
+    border: 5px solid rgb(0, 255, 8);
+    border-radius: 50%;
+    height: 10px;
+    top: 50%;
+    z-index: 1;
+  }
+
+  &__sweep-hand {
```

Comment:
This and the class below are very similar, could be done using `mixin` with passing some of the property values as arguments